tuxemon/core/components/event/actions/combat.py

```diff
@@ -213,6 +213,9 @@ def random_encounter(self, game, action):
         if not self.check_battle_legal(player1):
             return False
 
+        #stop player motion
+        player1.moving = False
```

Comment:
That could work for now, but it might cause a slightly jarring snapping of the character into place on the tile when we do the rounding. If we do the rounding just before the world state comes back after combat, it might remove that jarring effect. 

The snapping back effect would happen because, when you're holding down an arrow key, your character will keep moving past the tile to make the movement appear fluid.